src/libraries/System.Collections.Immutable/ref/System.Collections.Immutable.cs

```diff
@@ -375,14 +377,14 @@ public sealed partial class Builder : System.Collections.Generic.ICollection<Sys
             void System.Collections.Generic.ICollection<System.Collections.Generic.KeyValuePair<TKey,TValue>>.CopyTo(System.Collections.Generic.KeyValuePair<TKey, TValue>[] array, int arrayIndex) { }
             System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<TKey, TValue>> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<TKey,TValue>>.GetEnumerator() { throw null; }
             void System.Collections.ICollection.CopyTo(System.Array array, int arrayIndex) { }
-            void System.Collections.IDictionary.Add(object key, object value) { }
+            void System.Collections.IDictionary.Add(object key, object? value) { }
             bool System.Collections.IDictionary.Contains(object key) { throw null; }
             System.Collections.IDictionaryEnumerator System.Collections.IDictionary.GetEnumerator() { throw null; }
             void System.Collections.IDictionary.Remove(object key) { }
             System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
             public System.Collections.Immutable.ImmutableDictionary<TKey, TValue> ToImmutable() { throw null; }
             public bool TryGetKey(TKey equalKey, out TKey actualKey) { throw null; }
```

Comment:
Does this need [MaybeNullWhen(false)] on the out?

Reply:
The implementation for all kind of same, if actualKey not found `actualKey = equalKey` and `return false;`
So I thought as they are same TKey and will not me nullable (even if it was nullable both will be nullable)

Reply:
Ok.